app/views/schools/advice/storage_heaters/_holiday_usage_table.html.erb

```diff
@@ -0,0 +1,25 @@
+<table class="table advice-table">
+  <thead>
+    <tr>
+      <th>Holiday</th>
+      <th>Period</th>
```

Comment:
I think "Holiday" & "Period" may need extracting to translation yml files?

Reply:
OK, these are in the electricity and gas PRs too. I'll do a separate PR to fix that in all three